compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So people in other environments can't use our tools unless they upgrade the OS (or install Ubuntu OS).
+
+Therefore, we developed `onecc` that runs using a docker so that users can use `one-cmds` more easily. This is `onecc-docker.`
+
+> Currently, `onecc-docker` is only supported in Linux environments.
+
+
+
+## Requirements
+
+- Linux OS
+
+- Docker
+
+    - `onecc-docker` requires the current `user ID` to be included in the `docker group` because it requires the docker-related commands to be executed without `sudo` privileges.
+
+        ```
+        sudo usermod -aG docker ${USER}
+        ```
+
+- Python 3.8
+
+
+
+## Structure
+
+### Summary 
+
+`onecc-docker` creates a docker image with the latest release version of `ONE`, runs `onecc` inside the docker container after the container is driven to deliver the desired result to the user.
+
+### To check the latest version of ONE
+
+- We use github api to get the latest release version of ONE. Create a docker file using the imported version.
+
+### Check Docker Image Presence and Build Image
+
+- To prevent repetitive image generation, we verify that there is an `onecc` image of the latest `one` release version.
+- If not, build the docker image using the docker file.
```

Comment:
`build` -> `rebuild` ?

Reply:
In my opinion, we build the docker image only once (unless the user intentionally deletes the image), what we repeat is considered to be the creation and deletion of the container. 
@jyoungyun  
What do you think?

Reply:
> `build` -> `rebuild` ?

@seanshpark I can't catch your intention. :( Could you explain more?
The docker image is built only once there is no image with the same name. So I think `build` is more suitable for this step...

Reply:
Well, this maybe because I'm not a good Docker user/developer.
What I see is a Docker image. from the explanation, when the `latest` is changed, it sounds like Docker image is built again. So from the though of `again`, it looks like `rebuild`.
Technically it looks like `build` not `rebuild` so, we can go with `build`